packages/vx-brush/src/BaseBrush.tsx

```diff
@@ -89,6 +89,14 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     clickSensitivity: 200,
     resetOnEnd: false,
   };
+      
+  componentDidMount = () => {
+    window.addEventListener("mouseup", this.handleDragEnd);
```

Comment:
this will result in `handleDragEnd` being invoked (and subsequently `this.props.onChange/onBrushEnd` invocation and a `state` update) for `mouseup` events that have nothing to do with the `Brush` component. 

it's probably easiest to add a condition check in `this.handleDragEnd` that returns early / does nothing if `this.state.isBrushing` is `false`